app/models/ems_event.rb

```diff
@@ -283,10 +290,6 @@ def self.publish_event(ems_id, event)
 
   private_class_method :publish_event
 
-  private_class_method def self.syndicate_events?
-    Settings.event_streams.syndicate_events && MiqQueue.messaging_type != "miq_queue"
-  end
-
```

Comment:
IMHO, this method should be removed when the only mechanism for events is Kafka.

Reply:
:+1: